docs/_config.yml

```diff
@@ -1,4 +1,6 @@
-title: "AB-3"
+# @format
+
+title: "ClickConnect"
 theme: jekyll-theme-cayman
```

Comment:
Would it be good to update the theme to `minima`?

```suggestion
theme: jekyll-theme-minima
```

Reply:
Right! Good idea